src/libs/scene_utils/cesium/getMagic.ts

```diff
@@ -0,0 +1,9 @@
+import { defaultValue } from 'cesium';
+
+/**
+ * @private
+ */
+export function getMagic(tileBuffer: Buffer, byteOffset?: number) {
```

Comment:
maybe a more descriptive name than "getMagic" ?

Reply:
Had to dig a little bit to see what "magic" is referring to since this was copied from Cesium's Core library. Looks like it's just the tileset format so I'll update the names everywhere.